modules/nw-udn-benefits.adoc

```diff
@@ -0,0 +1,32 @@
+//module included in the following assembly:
+//
+// *networking/multiple_networks/about-user-defined-networks.adoc
+:_mod-docs-content-type: REFERENCE
+[id="nw-udn-benefits_{context}"]
+= Benefits of a user-defined network
+
+User-defined networks provide the following benefits:
```

Comment:
@tssurya this content comes from https://docs.openshift.com/container-platform/4.17/networking/multiple_networks/understanding-user-defined-network.html#:~:text=security%2C%20and%20performance.-,User,-%2Ddefined%20networks%20provide and is not new.